packages/react-component-library/src/components/Dropdown/Dropdown.tsx

```diff
@@ -71,7 +71,7 @@ const StyledSelect = styled(Select)`
   }
 
   .rn-dropdown__menu {
-    @include m.shadow('1');
+    box-shadow: ${shadow('1')};
```

Comment:
This one is a bit debatable; `shadow('1')` is obviously what was intended at some point, but it's barely noticeable and the default react-select shadow has been there for a while.